### DISCUSSION TITLE
OCCLLVM

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#984] [Ideas]
Compiles to LLVM IR. Use `llc` to compile to binary.

**Comment by GitMensch** [comment]
Sounds like a good idea, please feel free to work on that.

**Reply by ghost** [reply]
Don't need to troll me! I obviously don't have the ability to do that! But this is my solution for 64 bit support. At least, it could serve as a temporary solution until your own code generator fully support 64 bit.

**Comment by LADSoft** [comment]
im not sure what you are asking for here?

**Reply by GitMensch** [reply]
this is about generating LLVM intermediate and let llvm link it
